tests/fuzzer.c

```diff
@@ -1999,14 +1999,14 @@ static int basicUnitTests(U32 const seed, double compressibility)
 
     /* long rle test */
     {   size_t sampleSize = 0;
+        size_t expectedCompressedSize = 39; /* two RLE blocks, zstd 1.4.4 */
         DISPLAYLEVEL(3, "test%3i : Long RLE test : ", testNb++);
-        RDG_genBuffer(CNBuffer, sampleSize, compressibility, 0., seed+1);
```

Comment:
Correct me if I'm wrong, but it seems like neither of these `RDG_genBuffer()` actually do what we want? 

The first time it's called with `size = 0`, which means nothing gets generated. And I checked the compressed block that's the result of the second call, and it actually was not an RLE block. It seems like what we want to do is just generate two blocks of repeated characters, so it seems like we just want two memsets.

Reply:
You are completely right. This test doesn't help at all producing RLE blocks. It's no surprise it was ineffective in detecting RLE issues.